lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -57,14 +57,14 @@ def clear_shop_session
       session[:shopify_user] = nil
     end
 
-    def login_url
+    def login_url(no_cookie_redirect = false)
```

Comment:
Would be better if `no_cookie_redirect` was a [keyword argument](https://robots.thoughtbot.com/ruby-2-keyword-arguments).

Right now the API to call this method is: https://github.com/Shopify/shopify_app/pull/617/files#diff-350e7187d4b8bbcd1eaee3ff44fc7fb3R68
`login_url(true)`
while, with keyword argument, it would change to:
`login_url(no_cookie_redirect: true)`